packages/carbon-graphs/src/index.js

```diff
@@ -0,0 +1,6 @@
+import Carbon from './js/carbon';
+
+// TODO: update to pull scss from the src/lib folder instead of dist
+import '../dist/css/carbon-graphs.css';
```

Comment:
Note: This is a temporary fix to automatically import all necessary files. This will be replaces with scss files in the future.